sky/cli.py

```diff
@@ -992,6 +994,15 @@ def cli():
           'Setting this flag is equivalent to '
           'running ``sky launch -d ...`` and then ``sky autostop -i <minutes>``'
           '. If not set, the cluster will not be auto-stopped.'))
+@click.option(
+    '--terminate',
```

Comment:
How do you feel about the alternative of `--rm`? (E.g., use this name in user interfaces; in the backend we can keep `terminate`)

Pros:
- Popularized by docker: https://docs.docker.com/engine/reference/run/#clean-up---rm
- For some reason `--terminate` to me makes sense only for serverful settings. I can imagine in the future `sky launch` may gain the ability to launch serverless jobs. In that case, "--terminate a job after it exits" is a bit confusing imo.
  - Feel free to push back as (1) this seems a weak argument (2) `--rm` may not be that much better.
- Faster to type.

---

Another alternative is `--down`.

Pros:
- Matches `sky down`.
- Matches the colloquial "autodown".
- Faster to type.

Reply:
My 2c: - `--down` >> `--terminate` >> `--rm`

`--down` gets highest preference because its consistent with `sky down`. I'm a little apprehensive of `--terminate` mainly because it adds a new word `terminate` to user vocabulary, which has the same effect as `sky down`. Since `down` is a fundamental operation, I think it's a good idea to maintain consistency here.

Reply:
`--down` sounds great.

Minor: I see that some user-facing help strings get "terminate" replaced with "tear down". That seems okay but the former is much more common in cloud providers parlance (e.g., AWS console).

Reply:
Good point! I found we are using both `tear down` and `terminate` in our docs. I would prefer `tear down`, since it is closer to the `--down` or `sky down`. I went through the docstr's, and tried to make them more consistent, but still keep some of the `terminate`. PTAL. : )